docs/cli.rst

```diff
@@ -143,6 +143,9 @@ Example
     # Authenticate with Twitch
     twitch-oauth-token=mytoken
 
+.. note::
+    Full player paths are supported via configuration file options such as
+    ``player="C:\mpv-x86_64\mpv"``
```

Comment:
Line breaks don't work like this in ReST. You have to define `|br|` at the top of the document like in install.rst:
https://github.com/streamlink/streamlink/blame/0.14.2/docs/install.rst#L3-L6
And then use `|br|` like an HTML line break element:
https://github.com/streamlink/streamlink/blame/0.14.2/docs/install.rst#L293

Reply:
I don't actually want there to be a line break in the note there, it was just over the 79 character line length limit.